ngrinder-controller/src/main/java/org/ngrinder/security/PluggablePreAuthFilter.java

```diff
@@ -37,21 +37,14 @@
  * @author JunHo Yoon
  * @since 3.0
  */
-@Profile("production")
-@Component("pluggablePreAuthFilter")
 public class PluggablePreAuthFilter implements Filter {
-	@Autowired
-	private PluginManager pluginManager;
```

Comment:
Why do u remove Auth filter from spring component?

Reply:
Previous author not use DI for this class.

https://github.com/naver/ngrinder/blob/feature/convert-to-springboot/ngrinder-controller/src/main/java/org/ngrinder/infra/config/SecurityConfig.java#L273

So, I remove this class from root context beans. Maybe there is a reason didn't do DI. 

I think It can use DI because this object not change it's internal information anywhere.
I will change this and test it.